data/bundles/merkyl/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012, Marcel Hellkamp.
```

Comment:
Can we change the name of this file to LICENSE-bottle and update the header in the bottle file? As merkyl is actually written by us, and are just including bottle for convenience here?